docs/guide/cache/fastly.md

```diff
@@ -0,0 +1,470 @@
+# Configure and customize Fastly
+
+You can configure Fastly by using API calls or through the Fastly Web Interface. 
+Fastly provides a [Fastly CLI](https://developer.fastly.com/reference/cli/) for configuring Fastly through its API.
+
+Ibexa Cloud is delivered with Fastly preconfigured. 
+It means that you don't have to do any changes to the Fastly configuration in order to make your site work. 
+The information provided here is only applicable if you want to change the default Fastly configuration on Ibexa Cloud, 
+or if you are not using Ibexa Cloud and want to configure Fastly to work with [[= product_name =]] on premise.
+
+!!! note "The Fastly Web Interface is not available for Ibexa Cloud"
+    It's recommend for Ibexa Cloud customers to use the Fastly CLI instead of using the Fastly API directly with `curl`, and so on.
+
+
+## Getting Fastly credentials from Ibexa Cloud installation
+
+In order to use Fastly CLI or Fastly API directly, you need to obtain the credentials for your site.
+To obtain the cridentials connect via SSH to your Fastly-enabled environment (for example, production or staging) and
+type the command:
+
+```bash
+declare|grep FASTLY
+FASTLY_KEY=...
+FASTLY_SERVICE_ID=...
+```
+
+Note that these credentials will be different for your production and staging environments. When configuring the
+Fastly CLI, use the credentials for the environment you want to change.
+
+You may also notice difference in environment variable names.
+[[= product_name =]] uses the `FASTLY_SERVICE_ID` and `FASTLY_KEY` variables but you may ecounter `FASTLY_SERVICE_ID` and `FASTLY_API_TOKEN` in your configuration.
+
+!!! note "Varnish needs to be disabled when using Fastly"
+    Varnish is automatically provisioned on Ibexa Cloud. Varnish needs to be disabled on all environments which use
+    Fastly. See [documentation on how to do that](https://docs.platform.sh/guides/ibexa/fastly.html).
+
+## Quick setup of Fastly to use with [[= product_name =]]
+
+Use commands below to install VCL configurtion required for running Fastly with [[= product_name =]].
+You also need to set up domains, https and origin configuration (not covered here).
+All commands are explained in detail later on this page:
```

Comment:
Instead of "later on this page" add a link to the anchor:

"All commands are explained in detail in the [Section title](#section_header) section."